docs/installation/environment-variables.md

```diff
@@ -142,87 +142,10 @@ Configure the following environment variable to enable MicroPowerManager to send
-| `AIRTEL_IPS`         | `[]`                               | List of IP whitelisted for Airtel services. |
-
-#### Calin STS Meters
-
-For detailed information see [Calin Meter Developer Documentation](https://www.szcalinmeter.com/).
-
-| Environment Variable | Default                            | Description                                              |
-| -------------------- | ---------------------------------- | -------------------------------------------------------- |
-| `CALIN_CLIENT_URL`   | **Required** (when plugin is used) | Calin Meter client URL used for generating STS tokens.   |
-| `CALIN_USER_ID`      | **Required** (when plugin is used) | Calin Meter API username used for generating STS tokens. |
-| `CALIN_KEY`          | **Required** (when plugin is used) | Calin Meter API key used for generating STS tokens.      |
-
-If you meters are used which can send their consumption data to Calin's
-API the following environment variables need to be set.
-
-| Environment Variable | Default                            | Description                                                |
-| -------------------- | ---------------------------------- | ---------------------------------------------------------- |
-| `METER_DATA_URL`     | **Required** (when plugin is used) | Calin Meter API URL used for consumption data upload.      |
-| `METER_DATA_KEY`     | **Required** (when plugin is used) | Calin Meter API key used for consumption data upload.      |
-| `METER_DATA_USER`    | **Required** (when plugin is used) | Calin Meter API username used for consumption data upload. |
-
-#### Calin Smart Meters
-
-For detailed information see [Calin Smart Meter Developer Documentation](https://www.szcalinmeter.com/).
-
-| Environment Variable           | Default                            | Description                                                               |
-| ------------------------------ | ---------------------------------- | ------------------------------------------------------------------------- |
-| `CALIN_SMART_COMPANY_NAME`     | **Required** (when plugin is used) | Calin Smart Meter company name used for communication with Calin API.     |
-| `CALIN_SMART_PURCHASE_API_URL` | **Required** (when plugin is used) | Calin Smart Meter Purchase API URL used for communication with Calin API. |
-| `CALIN_SMART_CLEAR_API_URL`    | **Required** (when plugin is used) | Calin Smart Meter Clear API URL used for communication with Calin API.    |
-| `CALIN_SMART_USER_NAME`        | **Required** (when plugin is used) | Calin Smart Meter username used for communication with Calin API.         |
-| `CALIN_SMART_PASSWORD`         | **Required** (when plugin is used) | Calin Smart Meter password used for communication with Calin API.         |
-| `CALIN_SMART_PASSWORD_VENT`    | **Required** (when plugin is used) | Calin Smart Meter password vent used for communication with Calin API.    |
```

Comment:
Please also remove the corresponding section in `services.php`. https://github.com/EnAccess/micropowermanager/blob/main/src/backend/config/services.php#L55-L64

Reply:
CalinReadMeter.php is not using. We have plugins for Calin & Calin Smart meters. The code sample you shared me is legacy code from very old versions of MPM. So that can be removed.